testsuite/openmodelica/fmi/ModelExchange/2.0/testBug5673.mos

```diff
@@ -0,0 +1,256 @@
+// name:  testBug5673
+// keywords: FMI 2.0 export
+// status: correct
+// teardown_command: rm -rf binaries sources modelDescription.xml output.log
+
+loadString("
+package aliasCheck
+
+model test1
+ Real a, b=1;
+equation
+ a=b;
+end test1;
+
+model test2
+  parameter Real b=1;
+  Real a;
+equation
+  a=b;
+end test2;
+
+model test3
+  constant Real b=1;
+  Real a;
+equation
+  a=b;
+end test3;
```

Comment:
Don't clutter the test case by putting several models into it which aren't even referenced.